src/components/Unlock.vue

```diff
@@ -379,24 +379,24 @@
 					} else {
 						try_autounlock()
 					}
-				}).catch(err => {
+				}).catch(() => {
 					//this is fine - it just means the cache expired.  Clear the cache to be sure.
 					this.secureCache.clear('secureCache.entries')
 					try_autounlock()
-				}).then(nil => {
+				}).then(() => {
 					// state settled
 					this.busy = false
 					focus()
 				})
 			}
 
 			// modify unlockedState internal state
-			this.unlockedState.getTabDetails().then(nil => {
+			this.unlockedState.getTabDetails().then(() => {
 				if (this.unlockedState.sitePermission){
 					this.generalMessages.success = "You have previously granted Tusk permission to fill passwords on " + this.unlockedState.origin
-				}
-				else
-					this.generalMessages.warn = "This may be a new site to Tusk. Before filling in a password, double check that this is the correct site."
+				} else if (this.isUnlocked()) {
```

Comment:
This doesn't work - it's a race condition that will almost always fail.  unlocked state isn't set at the point that this executes.